modules/crud/sql/sql.go

```diff
@@ -77,6 +66,38 @@ func (s *SQL) GetDBType() utils.DBType {
 	return utils.MySQL
 }
 
+// IsClientSafe checks whether database is enabled and connected
+func (s *SQL) IsClientSafe() error {
+	if !s.enabled {
+		return utils.ErrDatabaseDisabled
+	}
+
+	if s.client == nil {
+		if err := s.connect(); err != nil {
+			log.Println("Error connecting to " + s.dbType + " : " + err.Error())
+			return utils.ErrDatabaseConnection
+		}
+	}
+
+	return nil
+}
+
+func (s *SQL) connect() error {
+	timeOut := 5 * time.Second
+
+	ctx, cancel := context.WithTimeout(context.Background(), timeOut)
+	defer cancel()
+
+	sql, err := sqlx.Open(s.dbType, s.connection)
```

Comment:
The `dbType` will have values like `sql-postgres` & `sql-mysql` I guess. You can't pass those directly I guess